src/api/install.ts

```diff
@@ -325,7 +317,10 @@ async function installInContext (
     logger.warn('`node_modules` is present. Shrinkwrap only installation will make it out-of-date')
   }
 
-  const nodeModulesPath = path.join(ctx.root, 'node_modules')
+  let nodeModulesPath = path.join(ctx.root, 'node_modules')
+  if (await fs.exists(nodeModulesPath)) {
+    nodeModulesPath = await fs.realpath(nodeModulesPath)
```

Comment:
the correct way to do it is put `fs.realpath` into a try/catch and on ENOENT use just `path.join`

```ts
function safeRealpath (filename: string) {
  try {
    return await fs.realpath(filename)
  } catch (err) {
    if (err['code'] === 'ENOENT') return filename
    throw err
  }
}
```

Reply:
the reason for that is because between calls to exists and realpath the directory could've been removed